forge-game/src/main/java/forge/game/ability/effects/ChooseGenericEffect.java

```diff
@@ -49,6 +50,8 @@ public void resolve(SpellAbility sa) {
         final int amount = AbilityUtils.calculateAmount(host, sa.getParamOrDefault("ChoiceAmount", "1"), sa);
         
         final boolean tempRem = sa.hasParam("TempRemember");
+        final boolean secretly = sa.hasParam("Secretly");
```

Comment:
should be "Secret" for consistency with other

Reply:
ChooseTypeEffect and ChoosePlayerEffect also had "Secretly" so I just changed VoteEffect